### PR TITLE
fix some punctuation in index.html

### PR DESCRIPTION
**Pull request by TallTed**
* put mdashes where they belong
* put commas after `e.g.`
* wrap a thing or two in backticks (which ReSpec will turn into `<code />` wrappers)

### CODE REVIEW

**Review by bblfish — APPROVED** [review]
looks good

**Review by csarven — CHANGES_REQUESTED** [review]
Document uses and intended to be available with UTF-8 encoding, and as per W3C, it is preferable to represent characters in their natural form, rather than escaping with character references. It is also generally more readable and maintainable. Suggesting `—` instead of `&mdash;`

File: spec/identity/index.html [hunk 1]
```diff
@@ -736,7 +736,7 @@ <h2 id="h2_references"><span class="secno">A. </span>References</h2><div id="nor
...
 </dd><dt id="bib-RDF-SYNTAX-GRAMMAR">[RDF-SYNTAX-GRAMMAR]</dt><dd rel="dcterms:requires">Fabien Gandon; Guus Schreiber. <a href="http://www.w3.org/TR/rdf-syntax-grammar/"><cite>RDF 1.1 XML Syntax</cite></a>. 9 January 2014. W3C Proposed Edited Recommendation. URL: <a href="http://www.w3.org/TR/rdf-syntax-grammar/">http://www.w3.org/TR/rdf-syntax-grammar/</a>
-</dd><dt id="bib-RDFA-CORE">[RDFA-CORE]</dt><dd rel="dcterms:requires">Ben Adida; Mark Birbeck; Shane McCarron; Ivan Herman et al. <a href="http://www.w3.org/TR/rdfa-core/"><cite>RDFa Core 1.1 - Second Edition</cite></a>. 22 August 2013. W3C Recommendation. URL: <a href="http://www.w3.org/TR/rdfa-core/">http://www.w3.org/TR/rdfa-core/</a>
+</dd><dt id="bib-RDFA-CORE">[RDFA-CORE]</dt><dd rel="dcterms:requires">Ben Adida; Mark Birbeck; Shane McCarron; Ivan Herman et al. <a href="http://www.w3.org/TR/rdfa-core/"><cite>RDFa Core 1.1 &mdash; Second Edition</cite></a>. 22 August 2013. W3C Recommendation. URL: <a href="http://www.w3.org/TR/rdfa-core/">http://www.w3.org/TR/rdfa-core/</a>
```
[the remark this answers is not in the document]
Comment (csarven) [reply spec/identity/index.html:739]: I'm not sure what GitHub will do with this suggested change given that the intention is to undo this bit. One way to find out.

```suggestion
</dd><dt id="bib-RDFA-CORE">[RDFA-CORE]</dt><dd rel="dcterms:requires">Ben Adida; Mark Birbeck; Shane McCarron; Ivan Herman et al. <a href="http://www.w3.org/TR/rdfa-core/"><cite>RDFa Core 1.1 - Second Edition</cite></a>. 22 August 2013. W3C Recommendation. URL: <a href="http://www.w3.org/TR/rdfa-core/">http://www.w3.org/TR/rdfa-core/</a>
```

[the remark this answers is not in the document]
Comment (TallTed) [reply spec/identity/index.html:739]: It is a regrettable truth that the source document used incorrect punctuation, in violation of the [W3C Manual of Style](https://www.w3.org/Guide/manual-of-style/), and its subsumed _The Chicago Manual of Style_ or _The Gregg Reference Manual_.

**Comment by csarven** [comment]
e.g.:

*Using character escapes in markup and CSS* describing "when not to use escapes" https://www.w3.org/International/questions/qa-escapes#not :

>It is almost always preferable to use an encoding that allows you to represent characters in their normal form, rather than using named character references or numeric character references.

*Character Model for the World Wide Web: Fundamentals* provides a common reference for text-manipulation, in particular a guideline for when to use escapes https://www.w3.org/TR/charmod/#C047 :

>Escapes SHOULD only be used when the characters to be expressed are not directly representable in the format or the character encoding of the document, or when the visual representation of the character is unclear.

More generally, the *Internationalization Best Practices for Spec Developers* https://www.w3.org/TR/international-specs/ informatively references *Character Model for the World Wide Web: Fundamentals*'s "character escaping" for basic guidelines related to the use of characters and encodings https://www.w3.org/TR/charmod/#sec-Escaping - among which is guideline C047 (linked above) although it is not explicitly called out in international-specs.

**Comment by TallTed** [comment]
@csarven — I've merged your `—` in place of my `&mdash;`, but I will also note that [WHEN ESCAPES CAN BE USEFUL](https://www.w3.org/International/questions/qa-escapes#use) (which follows [WHEN NOT TO USE ESCAPES](https://www.w3.org/International/questions/qa-escapes#not)) says that a "particularly useful role for escapes is to represent characters that are invisible or ambiguous in presentation."  I believe this applies to hyphen vs en-dash vs em-dash, particularly when working in spaces where monospaced fonts are common (e.g., source editors).  Reasonable minds may differ.

**Review by csarven — APPROVED** [review]

**Comment by bblfish** [comment]
If everyone's ok, I'll commit this PR soon.

**Pull request merged**